src/Registration.hs

```diff
@@ -1,5 +1,6 @@
 {-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE TemplateHaskell #-}
+{-# LANGUAGE ScopedTypeVariables #-}
```

Comment:
Not used?

Reply:
It's used on line 271 where we add a type hint on the _left_ hand side of an expression, because we don't actually use the variable.